### PR TITLE
React popper tooltip

### PR DESCRIPTION
**Pull request by dawinValenzuela**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [x] Bug fix (non-breaking change which fixes an issue)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**Storybook Link** (Link to a local storybook url to review new functionality)
https://storybook-react-popper-tooltip-7v6rokdl5q-uc.a.run.app/?path=/story/composer-primitive-tooltip--base

### CODE REVIEW

File: packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx [hunk 1]
```diff
@@ -1,111 +1,129 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
-import { Tooltip, TooltipPlace, TOOLTIP_DEFAULT_TEST_ID } from './Tooltip';
+import { render, screen, fireEvent, act, waitFor } from '@testing-library/react';
```
Comment (guilean) [inline packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx:2]: Is `act` used?

File: packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx [hunk 2]
```diff
@@ -1,111 +1,129 @@
...
+      />
+    );
+    const element = screen.getByText(trigger);
+    fireEvent.mouseEnter(element);
+    const tooltipElement = screen.getByTestId(TOOLTIP_DEFAULT_TEST_ID);
+    const arrow = tooltipElement.children[0];
+    await waitFor(() => expect(arrow.getAttribute('data-placement')).toEqual('right'));
   });
-});
+  it('should render backgroun color', async () => {
```
Comment (guilean) [inline packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx:113]: Typo -> backgroun**d**

File: packages/components/src/components/primitives/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-import colors from '../../../../lib/styles/colors.js';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { usePopper } from 'react-popper';
+import { getStyles } from './Tooltip.styles';
 
 export const TOOLTIP_DEFAULT_TEST_ID = 'tooltip';
 
-const defaultColor = colors['fill-primary-900'];
+export enum BackgroundColor {
```
Comment (guilean) [inline packages/components/src/components/primitives/Tooltip/Tooltip.tsx:8]: Rename to `TooltipBackgroundColor` to be consistent?

File: packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx (hunk 1 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx]: removed!

File: packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx (hunk 2 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx]: done!

File: packages/components/src/components/primitives/Tooltip/Tooltip.tsx (hunk 3 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Tooltip/Tooltip.tsx]: done!

**Pull request merged**